app/components/performance_table_helpers.rb

```diff
@@ -0,0 +1,24 @@
+module PerformanceTableHelpers
+  def trn_request_count_percent(attributes, numerator_key)
+    if attributes[:total].zero?
+      ""
+    else
+      percentage =
+        number_to_percentage(
+          100 * attributes[numerator_key].fdiv(attributes[:total]),
+          precision: 0
+        )
+      "(#{percentage})"
+    end
+  end
+
+  def number_with_percentage_cell(counts, key, label: nil)
+    [
+      number_with_delimiter(counts[key]),
```

Comment:
I discovered there's actually an ActiveSupport built-in with almost the same name that does the same thing https://api.rubyonrails.org/classes/ActiveSupport/NumberHelper.html#method-i-number_to_delimited

We should refactor away our custom one, not in this PR necessarily